src/com/puppetlabs/puppetdb/cli/services.clj

```diff
@@ -57,7 +57,9 @@
             [com.puppetlabs.puppetdb.config :as conf]
             [com.puppetlabs.puppetdb.utils :as utils]
             [puppetlabs.kitchensink.core :as kitchensink]
+            [robert.hooke :as rh]
             [puppetlabs.trapperkeeper.core :refer [defservice main]]
+            [puppetlabs.trapperkeeper.services :refer [service-id]]
```

Comment:
I believe you, but I can't for the life of me find the code in trapperkeeper that does this. It must been several macrons deep. In any case, I'll remove the import since it's misleading.